src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/NativeTypes.cs

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace System.DirectoryServices.Protocols
+{
+    internal static class AuthConstants
+    {
+        public const int SEC_WINNT_AUTH_IDENTITY_UNICODE = 0x2;
+        public const int SEC_WINNT_AUTH_IDENTITY_VERSION = 0x200;
+        public const string MICROSOFT_KERBEROS_NAME_W = "Kerberos";
+        public const string LDAP_SASL_AUTOMATIC = "0";
+    }
+
+    [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
```

Comment:
>CharSet = CharSet.Unicode [](start = 41, length = 25)

why this needed here?

Reply:
I'm actually not sure, this was the way this struct already existed, I just moved them to a new file.